config/settings/local.py

```diff
@@ -41,7 +41,7 @@
 # django-debug-toolbar
 # ------------------------------------------------------------------------------
 # https://django-debug-toolbar.readthedocs.io/en/latest/installation.html#prerequisites
-INSTALLED_APPS += ["debug_toolbar"]  # noqa F405
```

Comment:
Does ruff not use these?

Reply:
There's a check for noqa's that aren't used, and it flagged those. I'm wondering if these are redundant because we have `from .base import *  # noqa` at the top.